proc_install-knative-OCP-4x.adoc

```diff
@@ -0,0 +1,48 @@
+// This assembly is included in the following assemblies:
+//
+// assembly_knative-OCP-4x.adoc
+
+= Installing Knative on an OpenShift cluster 4.0 using the script provided
+
+NOTE: This Knative on OpenShift preview is only available by using the OpenShift 4.0 developer preview. 
+
+.Prerequisites
+* You will require a Red Hat Developers login to try this. 

```

Comment:
This is where try.openshift.com comes in - it's the prerequisite for using 4.0 clusters with Knative